src/renderer/components/NodeSelectorPanel/RepresentativeNodeWrapper.tsx

```diff
@@ -64,7 +63,13 @@ const RepresentativeNodeWrapper = memo(
                     closeOnMouseDown
                     hasArrow
                     borderRadius={8}
-                    label={<ReactMarkdown>{node.description}</ReactMarkdown>}
+                    label={
+                        <Box>
+                            <b>{collapsed ? node.name : ''}</b>
+                            {collapsed && <br />}
```

Comment:
Can be simplified.

(Suggestion needs formatting.)

```suggestion
                            {collapsed && <><b>{node.name}</b><br /></>}
```

Reply:
Ah, that's how you get it all into one thing. I tried something like that but it complained about multiple elements, didn't know I could wrap them in a single empty tag.